opentsdb/tsdb.go

```diff
@@ -2,6 +2,7 @@
 package opentsdb // import "bosun.org/opentsdb"
 
 import (
+	"bosun.org/cache"
```

Comment:
does that mean throwing out opentsdb's CacheContext ? graphite and opentsdb would each just have a simple context that does querying (and perhaps filtering and limiting) ?